packaging/ubuntu-14.04/rules

```diff
@@ -142,6 +144,10 @@ override_dh_install:
 	# and now the normal install rules
 	install --mode=0644 debian/snapd.system-shutdown.service debian/snapd/$(SYSTEMD_UNITS_DESTDIR)
 	$(MAKE) -C cmd install DESTDIR=$(CURDIR)/debian/tmp
+
+	# install any available upstart jobs
+	$(MAKE) -C data/upstart install DESTDIR=$(CURDIR)/debian/snapd/
```

Comment:
If you go with one makefile you could just roll that into the existing make install on data.